boards/nxp/mimxrt1180_evk/board.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/devicetree.h>
+#include <soc.h>
+#if defined(CONFIG_SOC_MIMXRT1189_CM7)
+#include <zephyr/cache.h>
+#elif defined(CONFIG_IMXRT118X_CM33_XCACHE_PS)
+#include <fsl_cache.h>
+#endif
+
+static int board_init_netc(void)
+{
+#if defined(CONFIG_ETH_NXP_IMX_NETC) && (DT_CHILD_NUM_STATUS_OKAY(DT_NODELABEL(netc)) != 0)
+/* TODO: support NETC with cache enablement */
```

Comment:
why completely disable the cache for the whole system here, instead of flushing/invalidating it in the driver? is it worse performance to do that than leave it disabled?

also, other users than the EVK will have to know to disable the cache or else have to debug a lot? it seems like this should be handled at the driver level. maybe make the driver in kconfig `depends on !(whichever cache related symbol, custom or common, might need a new one)`

Reply:
okay, let me re-consider how to handle this.
The reason disabling cache, is because we encountered some problem which need more investitation.
Enabling cache support for NETC will be next step.

Thanks.